go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/redhat-developer/gitops-operator
 go 1.16
 
 require (
-	github.com/argoproj-labs/argocd-operator v0.0.16-0.20210716115333-5b13ae4379ba
+	github.com/argoproj-labs/argocd-operator v0.0.16-0.20210722130040-5406a672631a
```

Comment:
Please update to pick up https://github.com/argoproj-labs/argocd-operator/pull/378 as well.

Reply:
Sure :)